plugins/woocommerce-admin/client/task-lists/fills/tax/stripe-tax/card.tsx

```diff
@@ -0,0 +1,105 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { getAdminLink } from '@woocommerce/settings';
+import { recordEvent } from '@woocommerce/tracks';
+import { Plugins } from '@woocommerce/components';
+import { dispatch, useDispatch } from '@wordpress/data';
+import { SETTINGS_STORE_NAME } from '@woocommerce/data';
+import { Button } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { PartnerCard } from '../components/partner-card';
+import { TaxChildProps } from '../utils';
+import StripeTaxLogo from './stripe-tax-logo.svg';
+import { createNoticesFromResponse } from '~/lib/notices';
+
+const STRIPE_TAX_PLUGIN_SLUG = 'stripe-tax-for-woocommerce';
+
+const redirectToStripeTaxSettings = () => {
+	window.location.href = getAdminLink(
+		'/admin.php?page=wc-settings&tab=stripe_tax_for_woocommerce'
+	);
+};
+
+export const Card: React.FC< TaxChildProps > = ( {
+	task: {
+		additionalData: { stripeTaxActivated } = {
+			stripeTaxActivated: false,
+		},
+	},
+} ) => {
+	const { createSuccessNotice } = useDispatch( 'core/notices' );
+
+	return (
+		<PartnerCard
+			name={ __( 'Stripe Tax', 'woocommerce' ) }
+			logo={ StripeTaxLogo }
+			description={ __( 'Powerful global tax tool', 'woocommerce' ) }
+			benefits={ [
+				__( 'Real-time sales tax calculation', 'woocommerce' ),
+				__( 'Multi-economic nexus compliance', 'woocommerce' ),
+				__( 'Detailed tax transaction reports', 'woocommerce' ),
+				__( 'Coverage in over 55 countries', 'woocommerce' ),
+				__( 'Free to install, then pay as you go.', 'woocommerce' ),
```

Comment:
Only this one has a period at the end. The others don't. Should we keep it consistent?